pyomo/contrib/mindtpy/tests/test_mindtpy_lp_nlp.py

```diff
@@ -23,7 +23,7 @@
 from pyomo.environ import SolverFactory, value
 from pyomo.opt import TerminationCondition
 
-required_solvers = ('ipopt', 'cplex_persistent')
+required_solvers = ('ipopt', 'cplex_persistent', 'gurobi_persistent')
```

Comment:
It might be the case one mip solver is available and the other one not. We should run the tests with whichever is available